phpunit.xml.dist

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit colors="true" bootstrap="vendor/autoload.php">
+<phpunit colors="true" bootstrap="vendor/autoload.php" stopOnFailure="true">
```

Comment:
Still please move it to `.travis.yml`. This would allow people checking out PR branch (for reviewing purposes) like me and using alternative PHPUnit runner not have this stopOnFailure behavior.